wagtail/contrib/redirects/middleware.py

```diff
@@ -17,11 +19,34 @@ def _get_redirect(request, path):
     site = Site.find_for_request(request)
     try:
         return models.Redirect.get_for_site(site).get(old_path=path)
+    except models.Redirect.DoesNotExist:
+        # Check for wildcard redirects
+        wildcard_redirects = models.Redirect.get_for_site(site).filter(
+            old_path__endswith="*"
+        )
+
+        for redirect in wildcard_redirects.iterator():
```

Comment:
@RealOrangeOne Would it work to have an option in the settings? For example, if the setting is set to true, then it will check for wildcard redirects; otherwise, it won't?